keyvault/data-plane/azure-keyvault-cryptography/pom.xml

```diff
@@ -1,100 +1,102 @@
-<!-- Copyright (c) Microsoft Corporation. All rights reserved. Licensed under 
-	the MIT License. See License.txt in the project root for license information. -->
+<!-- Copyright (c) Microsoft Corporation. All rights reserved. Licensed under
```

Comment:
Only thing changed in this file is indentation. GitHub isn't doing this diff well.